packages/exceptions/index.ts

```diff
@@ -0,0 +1,20 @@
+export * from './src/assert/AssertionException';
+export * from './src/base/Exception';
+export * from './src/base/RuntimeException';
+export * from './src/collections/NoSuchItemException';
+export * from './src/collections/IndexOutOfBoundsException';
+export * from './src/collections/EmptyQueueException';
+export * from './src/collections/EmptyStackException';
+export * from './src/format/FormatException';
+export * from './src/format/VersionFormatException';
+export * from './src/module/ClassLoaderException';
+export * from './src/text/ParsingException';
+export * from './src/uri/UriFormatException';
+export * from './src/uri/UriIntegrityException';
+export * from './src/InvalidArgumentException';
```

Comment:
Please put `InvalidArgumentException`, `InvalidOperationException`, `InvalidStateException`, `MethodNotImplementedException`, `MissingKeyException`, `OperationNotSupportedException` into `base` scope.

Reply:
WIll be reorganized after discussion and proposals